### PR TITLE
fix for #567 multiple return statements

### PR DESCRIPTION
**Pull request by moldover**
Fix for #567 multiple return statements.

### CODE REVIEW

File: src/main/java/network/brightspots/rcv/Tabulator.java [hunk 1]
```diff
@@ -394,27 +394,29 @@ private void setWinningThreshold(Map<String, BigDecimal> currentRoundCandidateTo
...
       // in this mode, we're done as soon as we've declared any winners
-      return numWinnersDeclared == 0;
+      keepTabulating = numWinnersDeclared == 0;
     } else {
       // If there are more seats to fill, we should keep going, of course.
       // But also: if we've selected all the winners in a multi-seat contest, we should tabulate one
       // extra round in order to show the effect of redistributing the final surpluses... unless
       // bottoms-up is enabled, in which case we can stop as soon as we've declared the winners.
-      return numWinnersDeclared < config.getNumberOfWinners()
+      keepTabulating = numWinnersDeclared < config.getNumberOfWinners()
```
Comment (HEdingfield) [inline src/main/java/network/brightspots/rcv/Tabulator.java:414]: It seems like you could just initially set it to this and get rid of the else statement, right? I guess it's more readable this way?

**Review by HEdingfield — APPROVED** [review]

File: src/main/java/network/brightspots/rcv/Tabulator.java (hunk 1 above)
Comment (tarheel) [reply src/main/java/network/brightspots/rcv/Tabulator.java:414]: I think it's more readable this way.

**Review by tarheel — APPROVED** [review]
Thanks! Did you confirm that these are the only offending methods via a manual audit, or did you use an automated check?

**Comment by moldover** [comment]
> Thanks! Did you confirm that these are the only offending methods via a manual audit, or did you use an automated check?

Manual audit.  It's not bad.

**Pull request merged**